include/Core/UI/Setting/CoreSetting.php

```diff
@@ -184,4 +186,30 @@ protected function get_user_roles( $option ) {
 
 		return $user_roles;
 	}
+
+	/**
+	 * Renders the Email Log `Remove Data on Uninstall?` settings.
+	 *
+	 * @param array $args
+	 */
+	public function render_dashboard_widget_status_settings( $args ) {
+		$option      = $this->get_value();
+		$remove_data = $option[ $args['id'] ];
```

Comment:
This variable name should be `hide_dashboard_widget`.

This looks like a copy/paste mistake. When you copy/paste code, make sure you make all the necessary changes.